app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -373,6 +374,15 @@ class TabTrayView(
             }
 
             layoutManager = gridLayoutManager
+
+            // Ensure items have the same all around border - 16 dp. Avoid the double spacing issue.
```

Comment:
I am gonna try rephrasing this comment so that there is more context around it, but I think we will need to touch up the explanation. I know you pointed me to https://stackoverflow.com/questions/28531996/android-recyclerview-gridlayoutmanager-column-spacing/28533234#28533234 regarding the issue:

> Set the correct column spacing in the grid layout to ensure a column spacing of 16dp between each grid item and avoid double spacing issue [TODO: I think the double spacing issue should be explained more clearly]. A 8dp padding is already set in tab_tray_grid_item.xml, so the parent needs to be padded with a remaining 8dp.

The point is to be pretty clear what this bit of code is doing and the problem it is solving. I know there's also the item decoration approach to solving the column spacing so I am wondering what was the difference between the 2 approaches.

Reply:
This to me seems simpler, easier to maintain and more performant since it's a one-time operation instead of setting new dimensions for each new item view.